transport/UdpEndpoint.cpp

```diff
@@ -26,21 +28,22 @@ class UnRegisterListenerJob : public jobmanager::Job
     Endpoint::IEvents* _listener;
 };
 
-class UnRegisterStunListenerJob : public jobmanager::Job
+class UnRegisterNotifyListenerJob : public jobmanager::Job
 {
 public:
-    UnRegisterStunListenerJob(UdpEndpoint& endpoint, __uint128_t transactionId)
+    UnRegisterNotifyListenerJob(UdpEndpoint& endpoint, Endpoint::IEvents& listener)
```

Comment:
Neatpicking: pass parameters as "const Type&". Though here methods are tiny and it does not worth changing. But I noticied a pattern.